src/code-examples/dialog/html.js

```diff
@@ -12,7 +12,7 @@ const html = `<button type="button" class="js-open-modal" aria-expanded="true" a
   <p>
     Bear claw sugar plum topping dessert ice cream gummi bears. Sweet roll apple pie pastry cookie croissant tootsie roll caramels cookie. Icing cheesecake chocolate cake tart chocolate gingerbread. Gummi bears ice cream apple pie cookie candy carrot cake fruitcake gummi bears marshmallow. Cotton candy pudding bonbon muffin jelly. Marshmallow chocolate cake cupcake muffin shortbread chocolate cake.
   </p>
-  <a href="#">Another element to demonstrate trapped focus :)</a>
+  <a href="#">Another element to demonstrate trapped focus 😃</a>
```

Comment:
💥 emoji are so accessible 🎉 🐹 🦑 ☕

Reply:
Do you just type them like normal? (control + command + space) Or do you use the unicodes?